style.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
 
-@media screen and (max-width:767px) {
+@media screen and (max-width: 767px) {
```

Comment:
- [ ] I think a major cause of the overflow problem. is that you didn't use a [mobile-first](https://zellwk.com/blog/how-to-write-mobile-first-css/) approach. It is a requirement for you to follow the [mobile-first](https://zellwk.com/blog/how-to-write-mobile-first-css/) approach [REQUIRED]